### PR TITLE
[heimdall] Added Heimdall Chart

### PR DESCRIPTION
**Pull request by skaro13**
Signed-off-by: Simon Caron <simon.caron@protonmail.com>

#### Special notes for your reviewer:

Added a new chart to deploy LinuxServer's Heimdall image.

#### Checklist
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/heimdall/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name my-release billimek/heimdall
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
```
Comment (billimek) [inline charts/heimdall/README.md:22]: With helm3 being the default now, we can probably drop the `--purge` flag for the `helm delete` command.

File: charts/heimdall/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+$ helm repo add billimek https://billimek.com/billimek-charts/
+$ helm install billimek/heimdall
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name my-release billimek/heimdall
```
Comment (billimek) [inline charts/heimdall/README.md:17]: With helm3 being the default now, we can probably drop the `--name` flag for the `helm install` command.

File: charts/heimdall/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+```console
+helm install --name my-release \
+  --set timezone="America/New York" \
+    billimek/heimdall
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+helm install --name my-release -f values.yaml stable/heimdall
```
Comment (billimek) [inline charts/heimdall/README.md:86]: This should be `billimek/heimdall`

**Comment by skaro13** [comment]
All fixed!

**Comment by billimek** [comment]
Thanks so much for contributing this chart, @skaro13!

**Pull request merged**